addons/website_blog/tests/common.py

```diff
@@ -35,3 +36,10 @@ def setUp(self):
             'notification_type': 'inbox',
             'groups_id': [(6, 0, [group_public_id])]
         })
+        self.user_portal = Users.with_context({'no_reset_password': True}).create({
+            'name': 'portal_user',
+            'login': 'portal_user',
+            'email': 'portal_user',
```

Comment:
Well if you write test, write them a bit completely ... use a real name, a real email on example.com, so that if necessary we can test other points.